docs/api/app.md

```diff
@@ -527,6 +527,151 @@ Adds `tasks` to the [Tasks][tasks] category of the JumpList on Windows.
+  standard `Tasks` category.
+* `"fileTypeRegistrationError"` - An attempt was made to add a file link to
+  the Jump List for a file type the app isn't registered to handle.
+* `"customCategoryAccessDeniedError"` - Custom categories can't be added to the
+  Jump List due to user privacy or group policy settings.
+
+If `categories` is `null` the previously set custom Jump List (if any) will be
+replaced by the standard Jump List for the app (managed by Windows).
+
+`JumpListCategory` objects should have the following properties:
+
+* `type` String - One of the following:
+  * `"tasks"` - Items in this category will be placed into the standard `Tasks`
+    category. There can be only one such category, and it will always be
+    displayed at the bottom of the Jump List.
+  * `"frequent"` - Displays a list of files frequently opened by the app, the
+    name of the category and its items are set by Windows.
+  * `"recent"` - Displays a list of files recently opened by the app, the name
+    of the category and its items are set by Windows. Items may be added to
+    this category indirectly using `app.addRecentDocument(path)`.
+  * `"custom"` - Displays tasks or file links, `name` must be set by the app.
+* `name` String - Must be set if `type` is `"custom"`, otherwise it should be
+  omitted.
+* `items` Array - Array of `JumpListItem` objects if `type` is `"tasks"` or
+          `"custom"`, otherwise it should be omitted.
+          
+**Note:** If a `JumpListCategory` object has neither the `type` nor the `name`
+property set then its `type` is assumed to be `"tasks"`. If the `name` property
+is set but the `type` property is omitted then the `type` is assumed to be
+`"custom"`.
+
+**Note:** Users can remove items from custom categories, and Windows will not
+allow a removed item to be added back into a custom category until **after**
+the next successful call to `app.setJumpList(categories)`. Any attempt to
+re-add a removed item to a custom category earlier than that will result in the
+entire custom category being omitted from the Jump List. The list of removed
+items can be obtained using `app.getJumpListSettings()`.
+
+`JumpListItem` objects should have the following properties:
+
+* `type` String - One of the following:
+  * `"task"` - A task will launch an app with specific arguments.
+  * `"separator"` - Can be used to separate items in the standard `Tasks`
+    category.
+  * `"file"` - A file link will open a file using the app that created the
+    Jump List, for this to work the app must be registered as a handler for
+    the file type (though it doesn't have to be the default handler).
+* `path` String - Path of the file to open, should only be set if `type` is
+  `"file"`.
+* `program` String - Path of the program to execute, usually you should
+  specify `process.execPath` which opens the current program. Should only be
+  set if `type` is `"task"`.
+* `arguments` String - The command line arguments when `program` is
```

Comment:
Can you rename this to `args`? It was a wrong decision to use `arguments` since it is a keyword of JavaScript and all other places are using `args`, it is good a chance to fix this in the new API.